app/models/manageiq/providers/openstack/cloud_manager/availability_zone.rb

```diff
@@ -6,7 +6,7 @@ class ManageIQ::Providers::Openstack::CloudManager::AvailabilityZone < ::Availab
   # hosts that has the matching availability_zone configured in /etc/cinder/cinder.conf.
   def block_storage_disk_capacity
     cluster = ext_management_system.provider.infra_ems.ems_clusters.find { |c| c.block_storage? == true }
-    cluster.aggregate_disk_capacity
+    cluster.nil? ? 0 : cluster.aggregate_disk_capacity
```

Comment:
I don't have a strong opinion on which style to use. If your suggestion is explicitly better based on MIQ coding style guidelines/preferences, I'll change it -- otherwise I'm inclined to just leave it as-is (both seem to be about equally readable, etc.)